lib/droppable.js

```diff
@@ -108,16 +124,20 @@ function uiDroppableController($scope, $element, $attrs, $parse) {
   function dragLeaveHandler(_event) {
     $element.removeClass(dropActiveClass);
     var dragHoverLeaveFn = $parse($attrs.onDragHoverLeave);
-    dragHoverLeaveFn($scope, {event: event});
+    dragHoverLeaveFn($scope, {
+      event: event
+    });
   }
 
   function _isAccepted(types) {
     for (var i = 0; i < types.length; i++) {
-      if (types[i] === accepts) {
-        return true;
+      for (var j = 0; j < accepts.length; j++) {
```

Comment:
http://stackoverflow.com/questions/16227197/compute-intersection-of-two-arrays-in-javascript

intersect

On Thu, Mar 3, 2016 at 8:21 PM, Robert Sebescen <notifications@github.com>
wrote:

> In lib/droppable.js
> <https://github.com/surikaterna/dragged/pull/1#discussion_r54932201>:
>
> >    }
> >
> >    function _isAccepted(types) {
> >      for (var i = 0; i < types.length; i++) {
> > -      if (types[i] === accepts) {
> > -        return true;
> > +      for (var j = 0; j < accepts.length; j++) {
>
> Lodash var inte en dependency av modulen så jag trodde att du ville hålla
> det vanilla js/bara blanda angular :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/surikaterna/dragged/pull/1/files#r54932201>.
>